docs/sdks/remove-api-key.md

```diff
@@ -31,11 +31,29 @@ from c8 import C8Client
 client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
                         email='nemo@nautilus.com', password='xxxxx',
                         geofabric='_system')
+                        
+# Get a list of all the API key ID's names
+apiKeyIds = []
+for api in client.list_all_api_keys():
+    apiKeyIds.append(api["keyid"])
 
-# Remove api key
-remove = client.remove_api_key('id1')
+# Check whether the chosen_api_key exists or not, comparing it to the API key list
+for apiElement in apiKeyIds:
+    if apiElement == CHOSEN_API_KEY_ID:
+        API_ACTIVE = True
+        break
 
-print(remove)
+# Checks the specified API key
+if API_ACTIVE:
+    # Removes the chosen API key
+    try:
+        print("Remove API key: ", client.remove_api_key(CHOSEN_API_KEY_ID))
+    except Exception as err:
+        print("Error removing the API key")
+        print(err)
+else:
+    # Displays a message that the key does not exist
+    print("Can't remove a non-existent key")
```

Comment:
```suggestion
    print("Can't remove a non-existent key.")
```

Reply:
Be sure to finish sentences with periods. Apply throughout.

Reply:
Thanks Diana.

This writing style is not in the original files.
Good to have it as a way to increase consistency on the docs.
Fixed.